### DISCUSSION TITLE
Unable to change directory

### DISCUSSION THREAD

**Discussion by lambdatheta-team** [community/community#48380] [Actions]
### Select Topic Area

Product Feedback

### Body

I am getting this errors as I just copied the code for creating war file of my eclipse project that is hosted on git now I am getting this errors and I am unable to know from where I should change this directory 

The goal you specified requires a project to execute but there is no POM in this directory (/home/runner/work/EAXee/EAXee). Please verify you invoked Maven from the correct directory. -> [Help 1]
Error:  
Error:  To see the full stack trace of the errors, re-run Maven with the -e switch.
Error:  Re-run Maven using the -X switch to enable full debug logging.
Error:  
Error:  For more information about the errors and possible solutions, please read the following articles:
Error:  [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MissingProjectException
Error: Process completed with exit code 1.

**Comment by jsoref** [comment]
It'd be really helpful if you provided a link to a public workflow in a public repository... And perhaps an indication of the directory you'd like to use...
